yang-module-versioning/draft-ietf-netmod-yang-module-versioning.txt

```diff
@@ -374,7 +374,8 @@ Internet-Draft    Updated YANG Module Revision Handling        June 2024
       semantics of the module.  Extension statement definitions SHOULD
       specify whether adding, removing, or changing statements defined
       by that extension are backwards-compatible or non-backwards-
-      compatible.
+      compatible, this is needed for clients which support that
+      extension.
```

Comment:
I'd recommend we split that "this is needed for clients" into its own sentence.

Reply:
Joe's comments are along the same lines, please see his suggestion (I think the comment was given on the .xml)